pkg/api/v1/types.go

```diff
@@ -1312,6 +1312,15 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty" description:"disable pod scheduling on the node; see http://releases.k8s.io/HEAD/docs/admin/node.md#manual-node-administration"`
 }
 
+type KubeletPorts struct {
+	// Port on which Kubelet is listening
+	KubeletPort uint `json:"kubeletPort" description:"port on which Kubelet is listening"`
+	// Read only port open on Kubelet
+	KubeletReadOnlyPorty uint `json:"kubeletReadOnlyPort" description:"read-only port exposed by Kubelet"`
```

Comment:
Added `omitempty`, as for it being used @roberthbailey commented on my WIP PR that depends on this chage https://github.com/kubernetes/kubernetes/pull/12919#discussion_r37452192. I know that we were using read only port for debugging, as it's way easier to curl it, instead of secured one. In other words: RO port future TBD, opened an issue #12968 to figure out what to do with it in the future. For now I'd leave it as it is, as the coming to the decision will probably take some time.